docs/documentation/aragon_network_token.md

```diff
@@ -0,0 +1,66 @@
+![](../images/aragon_network_token/ant.jpeg)
```

Comment:
Please use the image from `design/artwork/ANT` instead of adding bloat to the repo by adding the same image twice.